expected/copy-utility.source

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE customer
```

Comment:
This file was added just  to make directory non-empty, otherwise I failed to put in repository and without it regression test doesn't work,